PYME/LMVis/renderers.py

```diff
@@ -327,8 +327,9 @@ class Histogram3DRenderer(HistogramRenderer):
     mode = '3Dhistogram'
 
     def genIm(self, settings, imb, mdh):
-        mdh['Origin.z'] = settings['zBounds'][0]
```

Comment:
Why this change????? I think the old logic is doing the right thing

Reply:
I think `Origin.z` is only used in `PYME.IO.image` on line `344`. It looks like this is for backward compatibility. Do we need to create new images with `Origin.z` if all the new ones use are supposed to use `Camera` metadata?

Reply:
Definitely not just for backwards compatibility. image.origin is used when aligning images (e.g. making composites) either between different rendering modalities and/or different imaging modalities - e.g. a 3D reconstruction with a pre-acqisition widefield stack (potentially in a different colour channel). 

Because 3D PALM/STORM generates localizations above and below the current focal plane, and a reconstruction will naturally include extra planes for these, we can't just use the acquisition pifoc position, but need to record where our reconstruction actually starts.

Reply:
OK, that makes sense. Why do we have to choose `Origin.x` or `Camera.ROIPosX` then in `image.origin`? Why not just swap `Origin.x`'s value into `Camera.ROIPosX` and call it good?

Reply:
see reply below - replied to the notification email and it wound up in the wrong place

Reply:
Yeah, no worries. That makes sense. So after a rendering, `Camera.ROIPosX` is essentially just historical metadata. I think it would helpful to at least add some comments to `image.origin` explaining this. Better still, I might change `Origin.x` to `Renderer.Origin.x`.

Reply:
Ah, OK. I see it's used in cropping as well. So `Renderer.Origin.x` doesn't strictly make sense. Additional commentary in `image.origin` would suffice.